### ISSUE TITLE
Neomutt not showing latest state of IMAP folders

### ISSUE THREAD

**Issue by 5833tP2** [LukeSmithxyz/mutt-wizard#617]
I'm using two PCs to access my email: Thunderbird on one, and neomutt on the other (installed using mutt-wizard and running under arch). If I delete an In Box message with Thunderbird, I would expect that change to be shown in neomutt but this is not happening. I do not understand why, and now wonder whether I have a screwed up understanding of how neomutt is supposed to work.

My typical workflow is:

1. Start neomutt with `$ neomutt`
2. Update mailbox with `o` or `O`

New emails appear ok, but deleted emails are still visible. I now have an ever-longer list of emails in my In Box, even though they have been removed from the IMAP server.

What is happening? Or rather, what am I doing wrong?

**Comment by Kr1ss-XD** [comment]
- In your mbsync config (this is usually `~/.mbsyncrc`, but might be `~/.config/mbsyncrc`, depending on your environment) look for a section with `Expunge Both` and `Create Both` in it. This should be the `Channel` section.
- Add a line containing one of

```
Remove	Slave
```
or
```
Remove	Both
```

and sync you emails. After that, the deleted messages should be removed locally, too. If you choose `Both`, then messages you delete in neomutt will be removed from the IMAP server, too.

**Comment by 5833tP2** [comment]
My `~\.mbsyncrc` had only two lines (!)

```
SyncState *
Sync New ReNew Flags
```
I decided to delete (`mw -d me@mydomain.com`) then reinstall (`mw -a me@mydomain.com`) my email account. Now the `~\.mbsyncrc` file is much healthier. The Channel option is set as `Expunge Both` so I left that as it was and re-synced with the IMAP server. Success! The deleted emails were flagged with `D` in neomutt, and I purged them on exit.

I'll look up in the `man` pages about the difference between `Expunge Both` and `Remove Both` and decide which one will work better for me. In the meantime (and for the first time) neomutt is now working much more like I expected it to. I'm still getting used to it, but am much more optimistic now that syncing is working properly.

Thanks, @Kr1ss-XD for your detailed and helpful reply and solution to my problem.
